sdk/Lusid.Drive.Sdk.Tests/WaitForVirusScan.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Lusid.Drive.Sdk.Api;
+using Lusid.Drive.Sdk.Client;
+using Lusid.Drive.Sdk.Model;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using Polly;
+using Polly.Retry;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lusid.Drive.Sdk.Tests
+{
+    public class WaitForVirusScan
```

Comment:
this feels like it should be in the `Utilities` otherwise every user will end up with their own implementation

Reply:
Hey @rizwansaeed please can you re-review as I have now moved into `Utilities`.